katportalclient/test/test_client.py

```diff
@@ -786,13 +781,15 @@ def test_sensor_detail_exception_for_multiple_sensors(self):
     @gen_test
     def test_sensor_value_invalid_results(self):
         """test that we handle the monitor server returning an invalid string"""
+        yield self._portal_client._init_sitemap()   # Before we overwrite the mock fetch
```

Comment:
As discussed, would prefer not to do this as it will hide problems in the code under test which is expected to call this as necessary.

Reply:
Do we still need lines like this this in the tests?  Not all tests are doing it, so I wonder if it necessary.

Reply:
I'd intended to take them all out - not sure how they snuck back in. I'll take another look on Monday.